internal/dbtest/docker-compose.yaml

```diff
@@ -1,32 +1,32 @@
 services:
-  mysql8:
-    image: mysql:8.0
-    environment:
-      - MYSQL_DATABASE=test
-      - MYSQL_USER=user
-      - MYSQL_PASSWORD=pass
-      - MYSQL_ROOT_PASSWORD=pass
-    ports:
-      - 3306:3306
-    healthcheck:
-      test: ['CMD', 'mysqladmin', 'ping']
-      timeout: 5s
-      interval: 10s
-      retries: 3
-  mysql5:
-    image: mysql:5.7
-    environment:
-      - MYSQL_DATABASE=test
-      - MYSQL_USER=user
-      - MYSQL_PASSWORD=pass
-      - MYSQL_ROOT_PASSWORD=pass
-    ports:
-      - 53306:3306
-    healthcheck:
-      test: ['CMD', 'mysqladmin', 'ping']
-      timeout: 5s
-      interval: 10s
-      retries: 3
+  # mysql8:
```

Comment:
What is the purpose of this change? I don’t quite understand it for now.

Reply:
My bad! I commented those out for slightly faster test runs locally and must've committed those accidentally.
Thanks for catching it.